sortby.rb

```diff
@@ -0,0 +1,38 @@
+def bubble_sort_items(array)
+  items_unsorted = true
+  items_unsorted, array = sorting(array) while items_unsorted
+  array
+end
```

Comment:
What's this method for? You should only have two methods on this project: 
bubble_sort
bubble_sort_by
I recommend you to change the names to match those.
Also, delete the extra method and make sure you have only two that return the sorted array.